streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -36,7 +36,7 @@
  */
 public class StandbyTask extends AbstractTask {
 
-    private static final Logger log = LoggerFactory.getLogger(StandbyTask.class);
+    private final Logger log;
```

Comment:
Can we remove this and just use the one in `AbstractTask`?

Reply:
Will do that.